pkg/debezium/typeutil/helpers.go

```diff
@@ -706,13 +689,15 @@ func ExtractPostgresIntervalArray(interval string) ([]string, error) {
 	return result, nil
 }
 
-var timeWithoutTZ0 = "15:04:05"
-var timeWithoutTZ1 = "15:04:05.0"
-var timeWithoutTZ2 = "15:04:05.00"
-var timeWithoutTZ3 = "15:04:05.000"
-var timeWithoutTZ4 = "15:04:05.0000"
-var timeWithoutTZ5 = "15:04:05.00000"
-var timeWithoutTZ6 = "15:04:05.000000"
+var (
```

Comment:
this kind of changes - is a bit tricky, developers splitted into 2 main editors - vscode and goland, and first one automatically do left version and goland right version, at some point we decide to ignore this, so better to rollback all such changes.